cachito/web/config.py

```diff
@@ -64,7 +68,6 @@ class TestingConfig(DevelopmentConfig):
     # after the migration completes...
     #   https://github.com/miguelgrinberg/Flask-Migrate/issues/153
     SQLALCHEMY_DATABASE_URI = f"sqlite:///{TEST_DB_FILE}"
-    DEBUG = True
     LOGIN_DISABLED = False
```

Comment:
I believe we wanted to remove line with `LOGIN_DISABLED = False` because it's in the `Config` and keep line `DEBUG = True`.

Reply:
That's true, the `LOGIN_DISABLED` should be removed. The `DEBUG` was kept, though, he just moved it to the top.

Reply:
When I removed it, tests started failing

Reply:
> LOGIN_DISABLED = False because it's in the Config

But testing config inherits from `DevelopmentConfig` that requires `LOGIN_DISABLED = True`

Reply:
> > LOGIN_DISABLED = False because it's in the Config
> 
> But testing config inherits from `DevelopmentConfig` that requires `LOGIN_DISABLED = True`

My bad, I haven't noticed that.